src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -242,7 +242,7 @@ private function getDocBlockFromProperty($class, $property)
      * @param string $ucFirstProperty
      * @param int    $type
      *
-     * @return DocBlock|null
+     * @return array|null
```

Comment:
I think you forgot to expand the code :)

Look at the [line 287](https://github.com/symfony/symfony/pull/22251/files#diff-72b9199530f645acef692f3d77d4de4dR287)